index.js

```diff
@@ -1,5 +1,46 @@
 module.exports = decss
 
+/**
+ * @name decss
+ * @summary It generates components using passed CSS modules object.
```

Comment:
I think this should be in imperative (`Generate a component using passed CSS modules object.`) but I may be wrong here